### PR TITLE
Test cleanup: Move ephemeral volume test to unit tests

### PR DESCRIPTION
**Pull request by alromeros**
### What this PR does

This functional test only checks for converter behavior, which could easily be covered with unit tests.

This PR aims to move the test from functional to unit tests, in the same fashion as in the test here: https://github.com/kubevirt/kubevirt/pull/13051:

### Release note

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-launcher/virtwrap/converter/converter_test.go [hunk 1]
```diff
@@ -954,6 +954,51 @@ var _ = Describe("Converter", func() {
...
+								Ephemeral: &v1.EphemeralVolumeSource{
+									PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+										ClaimName: "pvc_block_test",
+									},
+								},
+							},
+						},
+					},
+				},
+			}
```
Comment (0xFelix) [inline pkg/virt-launcher/virtwrap/converter/converter_test.go:992]: Can you use `libvmi` for this please?

Comment (alromeros) [reply pkg/virt-launcher/virtwrap/converter/converter_test.go]: ack

File: pkg/virt-launcher/virtwrap/converter/converter_test.go [hunk 2]
```diff
@@ -954,6 +955,19 @@ var _ = Describe("Converter", func() {
 			Expect(*domain.Spec.Devices.Disks[0].Address).To(Equal(test_address))
 		})
 
+		It("should generate the block backingstore disk within the domain", func() {
+			vmi = libvmi.New(
+				libvmi.WithEphemeralPersistentVolumeClaim("pvc_block_test", "test-ephemeral"),
```
Comment (0xFelix) [inline pkg/virt-launcher/virtwrap/converter/converter_test.go:960]: Use const for `pvc_block_test`?

Comment (alromeros) [reply pkg/virt-launcher/virtwrap/converter/converter_test.go]: ack

File: pkg/virt-launcher/virtwrap/converter/converter_test.go [hunk 3]
```diff
@@ -80,6 +81,7 @@ const (
 	argMemBalloon10period = `<memballoon model="virtio-non-transitional" freePageReporting="on">
       <stats period="10"></stats>
     </memballoon>`
+	BlockPVCName = "pvc_block_test"
```
Comment (fossedihelm) [inline pkg/virt-launcher/virtwrap/converter/converter_test.go:84]: unexport this please `s/BlockPVCName/blockPVCName/`

**Review by fossedihelm — COMMENTED** [review]
Thank you! Just a small thing

File: pkg/virt-launcher/virtwrap/converter/converter_test.go (hunk 3 above)
Comment (alromeros) [reply pkg/virt-launcher/virtwrap/converter/converter_test.go]: Nice catch, ack

**Comment by fossedihelm** [comment]
Thank you! 
/lgtm

**Pull request merged**